.changeset/two-numbers-shake.md

```diff
@@ -0,0 +1,6 @@
+---
+"@twilio-paste/box": minor
+"@twilio-paste/core": minor
+---
+
+[Box] Add four new pseudo prop style selectors for use in the tabs package.
```

Comment:
suggestion: tell me what they are so I know to look for them

Reply:
done